package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "start": "slc run",
     "test": "mocha -R spec test",
-    "pretest": "jshint ."
+    "posttest": "eslint ."
```

Comment:
Talked to @superkhau: IBM has chosen `posttest` for the consideration of the external contributions.